internal/rbd/replicationcontrollerserver_test.go

```diff
@@ -64,7 +64,9 @@ func TestValidateSchedulingInterval(t *testing.T) {
 	}
 	for _, tt := range tests {
 		st := tt
+		tt := tt
```

Comment:
why this is needed? above `st:=tt` is already set? and why t.Parallel is defined two times in this testcase?

Reply:
@Madhu-1  parallelltest lint throws error if we have missing [t.Parallel in range method] (https://github.com/kunwardeep/paralleltest#missing-tparallel-in-the-range-method) as well. Aslo, it asks to reassign the range variables. though it doesn't make sense to reinitialize the variable, but it throws error if we don't.  Was waiting for the review on this. We can exclude this linter for such cases if we don't want to reinitialize the variable.

Reply:
`tt := tt` is not required? as `tt` is not used anywhere in the for loop?

Reply:
Yes, tt is not used anywhere in the loop, but since it is the loop variable according to the paralleltest lint we need to reinitialize it. We can also use tt instead of using st. Cause we can't avoid `tt := tt`

Reply:
then you can remove `st:= tt` and use `tt` wherever `st` is used?

Reply:
sure.